docs_guide/sphinx_guide/create_lists.rst

```diff
@@ -2,6 +2,8 @@
 Create lists
```

Comment:
Should this title should be changed to "How to create lists" like you did for other how-tos?

Reply:
Done in 129b3e07c368669d761a978193d8f4f63f250c22